libraries/botbuilder-azure/src/cosmosDbKeyEscape.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @module botbuilder-azure
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export module CosmosDBKeyEscape {
```

Comment:
>CosmosDBKeyEscape [](start = 14, length = 17)

I'm not sure why this needs to be a module versus just a set of exported functions.  At any rate it should be marked @private